### PR TITLE
V0.2.0

### PR DESCRIPTION
**Pull request by Guria**
cc @tadatuta

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Change Log
```
Comment (tadatuta) [inline CHANGELOG.md]: looks like `changelog` is one word (refer https://en.wikipedia.org/wiki/Changelog).

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## [Unreleased][unreleased]
+
+### Added
+
+- This Changelog
+
+### Changed
+
+- `bem-mod-*` attributes now treats value as expression.
```
Comment (tadatuta) [inline CHANGELOG.md]: treats -> treat (as `attributes` is plural form)

File: CHANGELOG.md (hunk 1 above)
Comment (Guria) [inline CHANGELOG.md]: Got template from http://keepachangelog.com/
But I don't mind to write in one word

File: README.md [hunk 3]
```diff
@@ -135,8 +135,11 @@ Support for explicit definition which block should be used is planned on near re
 
 ##### Binding to mods
 
-You could bind mods of `i-bem` blocks to scope variable with `bem-mod-mod-name="vm.modName"` syntax.
-Changing scope variable `vm.modName` will change value of mod `mod-name` and vice-versa.
+You could bind mods of `i-bem` blocks to expression result with `bem-mod-mod-name="expression"` syntax.
+`mod-name` would be updated with value `expression` evaluates to each time it changes.
+
+2 way binding to scope variable enabled with `bind:` prefix.
+Changing scope variable `vm.modName` will change value of mod `mod-name` and vice-versa when defined with `bem-mod-mod-name="vm.modName"` syntax.
```
Comment (tadatuta) [inline README.md]: vice-versa -> vice versa (refer https://ru.wikipedia.org/wiki/Vice_versa)

**Comment by tadatuta** [comment]
:+1:

File: CHANGELOG.md (hunk 2 above)
Comment (Guria) [inline CHANGELOG.md]: maybe "`bem-mod-*` attributes value now treated as expression." ?
attributes not independent. it processed in `bemDirective`

Comment (tadatuta) [inline CHANGELOG.md]: `bem-mod-*` attributes value **is** now treated as expression

**Pull request merged**